lib/srv/alpnproxy/auth/auth_proxy.go

```diff
@@ -113,13 +114,21 @@ func (s *AuthProxyDialerService) dialLocalAuthServer(ctx context.Context) (net.C
 	if len(authServers) == 0 {
 		return nil, trace.NotFound("empty auth servers list")
 	}
-	//TODO(smallinksy) Better support for HA. Add dial retry on auth network errors.
-	authServerIndex := rand.Intn(len(authServers))
-	conn, err := net.Dial("tcp", authServers[authServerIndex].GetAddr())
-	if err != nil {
-		return nil, trace.Wrap(err)
+	var errors []string
+
+	//select random server from list, try others if it's unavailable
```

Comment:
It's idiomatic to leave a space between the `//` and the comment if it's not a comment meant for automated tools:

```suggestion
	// iterate over the addresses in random order
```